frontend/discourse/app/components/user-fields/dropdown.gjs

```diff
@@ -7,14 +7,16 @@ import UserFieldBase from "./base";
 
 export default class UserFieldDropdown extends UserFieldBase {
   <template>
-    <label
-      class="control-label alt-placeholder"
-      for={{concat "user-" this.elementId}}
-    >
-      {{this.field.name}}
-      {{~#unless this.field.required}}
-        {{i18n "user_fields.optional"}}{{/unless~}}
-    </label>
+    {{#if @showLabel}}
```

Comment:
Something is janky with these fields, I'm getting double ups in labels etc:

<img width="440" height="802" alt="image" src="https://github.com/user-attachments/assets/c778a56f-fe99-4344-973c-feb1da890d3b" />

To test you need to add one via http://localhost:4200/admin/config/user-fields